auth-web/src/components/auth/manage-business/AffiliatedEntityList.vue

```diff
@@ -111,6 +170,15 @@ export default class AffiliatedEntityList extends Vue {
     return corpType === CorpType.NAME_REQUEST
   }
 
+  private isApprovedForIA (business: Business): boolean {
+    // Split string tokens into an array to avoid false string matching
+    const supportedEntityFlags = LaunchDarklyService.getFlag(LDFlags.IaSupportedEntities)?.split(' ')
+
+    return this.isNameRequest(business.corpType.code) &&
+      business.nameRequest?.state === NrState.APPROVED &&
```

Comment:
Is Conditionally Approved a value state for using a NR?

Reply:
I'm not sure, good question. Was not called out in the tickets🤷

Reply:
Will update this in the future if the requirements are updated 👍

Reply:
OK, I'll resolve this comment, but please ask BA if maybe this was a missed req.